tests/test_config.py

```diff
@@ -152,6 +152,19 @@ def test_from_pretrained_cache_dir_remote(self):
             PeftConfig.from_pretrained("ybelkada/test-st-lora", cache_dir=tmp_dirname)
             assert "models--ybelkada--test-st-lora" in os.listdir(tmp_dirname)
 
+    @parameterized.expand(ALL_CONFIG_CLASSES)
+    def test_save_pretrained_with_runtime_config(self, config_class):
+        r"""
+        Test if the config correctly removes runtime when saving
+        """
+        with tempfile.TemporaryDirectory() as tmp_dirname:
+            for model_name, revision in PEFT_MODELS_TO_TEST:
+                cfg = config_class.from_pretrained(model_name, revision=revision)
+                cfg.runtime.ephemeral_transfers = True
```

Comment:
I'm confused why this is not erroring since it should only work for `LoraConfig`.